packages/web-frontend/src/containers/MapOverlayBar/Pin.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Tupaia Web
+ * Copyright (c) 2021 Beyond Essential Systems Pty Ltd.
+ * This source code is licensed under the AGPL-3.0 license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { IconButton as MuiIconButton } from '@material-ui/core';
+import { DARK_GREY, TUPAIA_ORANGE } from '../../styles';
+
+const pathId = 'pin';
+const IconButton = styled(MuiIconButton)`
+  padding: 0;
+  #${pathId} {
+    fill: ${props => (props.$isPinned ? TUPAIA_ORANGE : DARK_GREY)};
+  }
+
+  &:hover {
+    #${pathId} {
+      fill: ${props => (props.$isPinned ? TUPAIA_ORANGE : 'white')};
+    }
+  }
```

Comment:
```suggestion
  path {
    fill: ${props => (props.$isPinned ? TUPAIA_ORANGE : DARK_GREY)};
  }

  &:hover {
    path {
      fill: ${props => (props.$isPinned ? TUPAIA_ORANGE : 'white')};
    }
  }
```

Reply:
This is optional but just letting you know that it is possible to select the path without needing to add an id.